app/screens/wallet/Backup.js

```diff
@@ -0,0 +1,187 @@
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+`;
+
+const SecurityLogo = styled.img`
+  height: 66px;
+  width: 54px;
+  margin-right: 30px;
+`;
+
+const BackupHeader = styled.span`
+  font-size: 24px;
+  font-weight: bold;
+  line-height: 33px;
+  color: ${smColors.darkGray};
+`;
+
+const BackupTitleWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+`;
+
+const Recommended = styled.div`
+  align-self: flex-end;
+  padding-left: 24px;
+`;
+
+type Props = {};
+
+class Backup extends Component<Props> {
+  render() {
+    return (
+      <Wrapper>
+        <Header>Wallet Backup</Header>
+        <BaseText>Your wallet file is encrypted with your passphrase on your computer, but We recommend that you’ll backup your wallet for additional security.</BaseText>
+        <Separator />
+        <BoldText>How would you like to backup your wallet?</BoldText>
+        <BackupBoxesWrapper>
+          {this.renderBackupBox('file')}
+          {this.renderBackupBox('mnemonic')}
+        </BackupBoxesWrapper>
+        <ActionLink onClick={this.learnMoreAboutSecurity}>Learn more about wallet security and backup</ActionLink>
+      </Wrapper>
+    );
+  }
+
+  renderBackupBox = (mode: 'file' | 'mnemonic') => {
+    const shieldLogo = mode === 'file' ? shieldIconGreenOne : shieldIconOrangeTwo;
+    const securityLevel = `${mode === 'file' ? 'Basic' : 'Enhanced'} Security`;
+    const backupMode = `${mode === 'file' ? 'File' : '12 Words'} Backup`;
+    const bodyText = {
+      file: 'Take your wallet file and save a copy of it in a secure location - We recommend a USB key or a external drive.',
+      mnemonic: 'A list of words you keep on a physical paper or in your password manager, that can be used to recover your wallet on any device.'
+    };
+    const noteText = {
+      file: 'NOTE: You will still need your passphrase to restore the wallet from the file.',
+      mnemonic: 'NOTE: Works even if you lost your passphrase or wallet backup file'
+    };
+    const linkText = mode === 'file' ? 'Show me the wallet file' : 'Create a paper backup';
+    const linkActions = {
+      file: this.showWalletFile,
+      mnemonic: this.createPaperBackup
+    };
+
+    return (
+      <BackupBox borderColor={mode === 'file' ? smColors.green : smColors.orange}>
+        <BackupTopWrapper>
+          <SecurityLogo src={shieldLogo} />
+          <BackupTitleWrapper>
+            <GrayText>{securityLevel}</GrayText>
+            <BackupHeader>{backupMode}</BackupHeader>
+          </BackupTitleWrapper>
+          {mode === 'mnemonic' && (
+            <Recommended>
+              <GrayText>Recommended</GrayText>
+            </Recommended>
+          )}
+        </BackupTopWrapper>
+        <BaseText>{bodyText[mode]}</BaseText>
+        <BaseText>{noteText[mode]}</BaseText>
+        <ActionLink onClick={linkActions[mode]} theme={mode === 'file' ? 'green' : 'orange'}>
```

Comment:
why not defining color here and making another evaluation in styled component?

Reply:
ok